pkg/trace/api/otlp.go

```diff
@@ -94,6 +89,13 @@ func (o *OTLPReceiver) Start() {
 				}
 			}()
 			log.Debugf("Listening to core Agent for OTLP traces on internal gRPC port (http://%s:%d, internal use only). Check core Agent logs for information on the OTLP ingest status.", cfg.BindHost, cfg.GRPCPort)
+
+			computeTopLevelBySpanKindVal := 0.0
+			if !o.conf.HasFeature("disable_otlp_compute_top_level_by_span_kind") {
+				log.Warn("OTLP ingest now identifies top-level spans by span kind by default, which may impact the number of spans that generate trace metrics. This new logic can be disabled if needed by adding 'disable_otlp_compute_top_level_by_span_kind' in DD_APM_FEATURES.")
```

Comment:
simplify 
```
OTLP ingest identifies top-level spans by span kind it may impact the trace metrics. The feature can be disabled by adding 'disable_otlp_compute_top_level_by_span_kind' in DD_APM_FEATURES
```